sycl/include/CL/sycl/multi_ptr.hpp

```diff
@@ -46,6 +46,13 @@ template <typename ElementType, access::address_space Space> class multi_ptr {
     // TODO An implementation should reject an argument if the deduced
     // address space is not compatible with Space.
   }
+#if defined(RESTRICT_WRITE_ACCESS_TO_CONSTANT_PTR)
+  template <access::address_space _Space = Space,
+            typename = typename std::enable_if<
+                _Space == Space &&
```

Comment:
This line is unnecessary. `_Space == Space` is guaranteed to be true (constructor template arguments can't be explicitly specified).

Reply:
Removed.